src/commands/infra/generate.test.ts

```diff
@@ -66,4 +68,31 @@ describe("Validate cloning of a remote repo from source", () => {
     expect(await validateRemoteSource(testValues)).toBe(true);
     // Need improved tests to check cloned repo
   });
+}); */
+
+describe("Validate template path from a definition.json", () => {
+  test("Validating that generate can extract a path from a definition.json file", async () => {
+    const mockProjectPath = "src/commands/infra/mocks";
+    const templatePath = await parseDefinitionJson(mockProjectPath);
+    expect(templatePath).toContain(
+      "_microsoft_bedrock_git/cluster/environments/azure-single-keyvault"
+    );
+  });
+});
+
+describe("Validate spk.tfvars file", () => {
+  test("Validating that a spk.tfvars is generated and has appropriate format", async () => {
+    const mockProjectPath = "src/commands/infra/mocks";
+    const generateTfvars = await generateSpkTfvars(
+      mockProjectPath,
+      mockProjectPath
+    );
+    const data = fs.readFileSync(
+      path.join(mockProjectPath, "spk.tfvars"),
+      "utf-8"
+    );
+    logger.info(data);
```

Comment:
Consider removing the log function if the intent of this was to help you debug initially.